src/components/Button/BaseButton.tsx

```diff
@@ -80,8 +81,10 @@ export const BaseButton: FC<InternalButtonProps> = React.forwardRef(
             { [styles.buttonSize2]: size === ButtonSize.Medium },
             { [styles.buttonSize3]: size === ButtonSize.Small },
             { [styles.pillShape]: shape === ButtonShape.Pill },
+            { [styles.roundShape]: shape === ButtonShape.Round },
```

Comment:
You'll need to handle this so that when `split` is true, and that `TwoStateButton` may not have a round shape. Those would look funky.

Reply:
Let's restrict round to buttons with icons only for now.